modules/apps/fragment/fragment-video-streaming/src/main/resources/META-INF/resources/page.jsp

```diff
@@ -0,0 +1,62 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<liferay-util:html-top
+	outputKey="vide_sctreaming_css"
+>
+	<link href="https://vjs.zencdn.net/8.6.1/video-js.min.css" rel="stylesheet" type="text/css" />
+</liferay-util:html-top>
+
+<div style="display: flex; justify-content: left;">
+	<div class="videojs-container">
+		<video class="video-js" controls id="fragmentVideoJsURL" preload="auto">
+			<source src="<%= (String)request.getAttribute("src") %>" type="video/mp4" />
+		</video>
+
+		<script src="https://vjs.zencdn.net/8.6.1/video.min.js"></script>
+	</div>
+</div>
+
+<script>
```

Comment:
I know this should be in a different .js file, but it was throwing errors, and as this is a POC, for now I decided to leave it this way and keep focusing on the needs for the RFP.